@here/harp-examples/src/effects.ts

```diff
@@ -0,0 +1,209 @@
+        // Instantiate the default map controls, allowing the user to pan around freely.
+        const mapControls = new MapControls(map);
+        mapControls.maxPitchAngle = 60;
+        mapControls.setRotation(6.3, 60);
+        // end:harp_gl_hello_world_example_2.ts
+
+        // Add an UI.
+        const ui = new MapControlsUI(mapControls);
+        canvas.parentElement!.appendChild(ui.domElement);
+
+        // snippet:harp_gl_hello_world_example_3.ts
+        // Resize the map to maximum.
+        map.resize(window.innerWidth, window.innerHeight);
+
+        // React on resize events.
+        window.addEventListener("resize", () => {
+            map.resize(window.innerWidth, window.innerHeight);
+        });
+        // end:harp_gl_hello_world_example_3.ts
+
+        return map;
+    }
+
+    const mapView = initializeMapView("mapCanvas");
+
+    const hereCopyrightInfo: CopyrightInfo = {
+        id: "here.com",
+        year: new Date().getFullYear(),
+        label: "HERE",
+        link: "https://legal.here.com/terms"
+    };
+    const copyrights: CopyrightInfo[] = [hereCopyrightInfo];
+
+    // snippet:harp_gl_hello_world_example_4.ts
+    const omvDataSource = new OmvDataSource({
+        baseUrl: "https://xyz.api.here.com/tiles/herebase.02",
+        apiFormat: APIFormat.XYZOMV,
+        styleSetName: "tilezen",
+        maxZoomLevel: 17,
+        authenticationCode: accessToken,
+        copyrightInfo: copyrights
+    });
+    // end:harp_gl_hello_world_example_4.ts
+
+    // snippet:harp_gl_hello_world_example_5.ts
+    mapView.addDataSource(omvDataSource);
+    // end:harp_gl_hello_world_example_5.ts
+
+    const gui = new GUI({ width: 300 });
+    const options = {
+        labels: false,
+        toneMappingExposure: 1.0,
+        outline: {
+            enabled: true,
+            ghostExtrudedPolygons: false,
+            thickness: 0.004,
+            color: "#898989"
+        },
+        bloom: {
+            enabled: true,
+            strength: 0.5,
+            threshold: 0.83,
+            radius: 1
+        },
+        vignette: {
+            enabled: true,
+            offset: 1.0,
+            darkness: 1.0
+        },
+        sepia: {
+            enabled: true,
+            amount: 0.55
+        }
+    };
+
+    mapView.addEventListener(MapViewEventNames.ThemeLoaded, () => {
+        mapView.renderLabels = options.labels;
+        mapView.mapRenderingManager.outline.enabled = options.outline.enabled;
+        mapView.mapRenderingManager.updateOutline(options.outline);
+        mapView.mapRenderingManager.bloom.enabled = options.bloom.enabled;
+        mapView.mapRenderingManager.bloom = options.bloom;
+        mapView.mapRenderingManager.vignette = options.vignette;
+        mapView.mapRenderingManager.sepia = options.sepia;
+        mapView.update();
+    });
+
+    gui.add(options, "labels").onChange((value: boolean) => {
```

Comment:
What about one function that changes all the effects from the `options` object and is reused everywhere, from the theme-loaded callback to the gui callbacks, it would clean the code.